pages/faq.js

```diff
@@ -0,0 +1,41 @@
+import styles from './faq.module.css'
+
```

Comment:
importing styles from the empty file? I guess this pr is only for contents and style comes later?

Reply:
ah yeah, i will add some styling, but mostly for content